sdk/core/core-http/lib/util/xml.browser.ts

```diff
@@ -147,3 +142,33 @@ function buildNode(obj: any, elementName: string): Node[] {
     throw new Error(`Illegal value passed to buildObject: ${obj}`);
   }
 }
+
+export function convertAtomXmlToJson(body: string): any {
+  const dom = parser.parseFromString(body, "text/xml");
+  throwIfError(dom);
+  const result = domToObject(dom);
+  return result;
+}
+
+/**
+ * @param {object} content The content payload as it is to be serialized. It should include any root node(s).
+ */
+export function convertJsonToAtomXml(content: any): string {
+  content[Constants.XML_METADATA_MARKER] = { type: "application/xml" };
+  const res = buildNode(content, "content");
+  const dom = res[0];
+
+  const result =
+    `<?xml version="1.0" encoding="utf-8" standalone="yes"?><entry xmlns="http://www.w3.org/2005/Atom"><updated>${new Date().toISOString()}</updated>` +
+    serializer.serializeToString(dom) +
+    `</entry>`;
+  return result;
+}
+
+export function stringifyXML(obj: any, opts?: { rootName?: string }) {
```

Comment:
Please move this function back to its original position.
Otherwise, when looking back using `git blame`, this commit will unnecessarily show up as affecting the `stringifyXML` function